src/components/[guild]/RolePlatforms/RolePlatforms.tsx

```diff
@@ -124,6 +139,45 @@ const RolePlatforms = ({ roleId }: Props) => {
                     )
                   }
                   actionRow={PlatformCardSettings && <PlatformCardSettings />}
+                  contentRow={
+                    CAPACITY_TIME_PLATFORMS.includes(type) && (
+                      <Wrap>
```

Comment:
Here's an external Wrap, and there's another wrap in CapacityTimeTags. Would be nice if both the tags and the edit button could be in a single Wrap